travis/install-tools.sh

```diff
@@ -31,6 +31,9 @@ else
     cp requirements-conda.txt $HOME
 fi
 
+# Set to conda's java
+# export JAVA_HOME="$(pwd)/tools/conda-tools/default/jre"
```

Comment:
So is this `export JAVA_HOME` statement commented out because it already happens anyway when we activate the conda environment?